demo/docker-compose.yaml

```diff
@@ -22,8 +22,8 @@ services:
       ]
 
   hydra-node-1:
-    # NOTE: Make sure to use the same image in ./seed-devnet.sh
-    image: ghcr.io/input-output-hk/hydra-node:0.11.0
+   # NOTE: Make sure to use the same image in ./seed-devnet.sh
+    image: ghcr.io/input-output-hk/hydra-node:unstable
```

Comment:
We would overwrite these hydra-node versions upon each release. I guess you needed to use the latest on master for the demo and also it should be ok to keep it as is here since 0.12 will update it.

Reply:
Correct